GVFS/GVFS.Virtualization/Projection/GitIndexProjection.cs

```diff
@@ -1431,22 +1446,18 @@ private string GetNewProjectedShaForPlaceholder(string path)
         /// <summary>
         /// Removes the folder placeholder from disk if it's empty.
         /// </summary>
-        /// <returns>
-        /// <c>true</c>If the folder placeholder was deleted
-        /// <c>false</c>If RemoveFolderPlaceholderIfEmpty failed attempting to remove the folder placeholder
+        /// <returns> Result of trying to delete the PlaceHolder
         /// </returns>
-        private bool RemoveFolderPlaceholderIfEmpty(IPlaceholderData placeholder)
+        private FSResult RemoveFolderPlaceholderIfEmpty(IPlaceholderData placeholder)
         {
             UpdateFailureReason failureReason = UpdateFailureReason.NoFailure;
             FileSystemResult result = this.fileSystemVirtualizer.DeleteFile(placeholder.Path, FolderPlaceholderDeleteFlags, out failureReason);
             switch (result.Result)

```

Comment:
This could be changed into an `if` statement to log and always return the `result.Result` but would have a bigger diff so I think this is good for now.